content/tracing/python.md

```diff
@@ -18,6 +18,11 @@ pip install ddtrace
 
 Finally, import the tracer and instrument your code!
 
+<div class="alert alert-warn">
+For Python applciations, please note that tracing is disabled when your application is launched in <b>DEBUG</b> mode.
+Find more <a href="http://pypi.datadoghq.com/trace/docs/#module-ddtrace.contrib.django">here</a>
```

Comment:
this sounds like something is missing. Find more what here?? perhaps "find more details here" or "learn more about this here" or something like that